src/locales/en.json

```diff
@@ -2155,6 +2155,7 @@
 "label.usageinterface": "Usage Interface",
 "label.usagename": "Usage Type",
 "label.usageunit": "Unit",
+"label.use.system.timezone": "Use System Timezone",
```

Comment:
As I said that was just nit-picking. This is actually more misleading (which system?), the previous one made more sense (I think most users may prefer Computer timezone or local timezone... consult with others?)